opendebates/templates/opendebates/list_ideas.html

```diff
@@ -79,6 +79,7 @@
 {% endblock %}
 
 {% block primary_content %}
+  {% cache 30 idea_list search_term category.id sort %}
```

Comment:
@ejucovy I think you're right, since the contents here will be rendered differently depending upon what page we're on.